packages/rum/src/types/record.ts

```diff
@@ -0,0 +1,236 @@
+import type { TimeStamp } from '@datadog/browser-core'
```

Comment:
I like the split of the types but now that I'm looking at it, it feels a bit weird to have `domain/record/types.ts` and `/types/record.ts`. 
Either we collocate the types to the module or use a `types` folder. My personal preference is to collocate the type with the module but it's not a strong opinion. 
Wdyt